modules/order/src/OrderStorage.php

```diff
@@ -44,6 +51,7 @@ class OrderStorage extends CommerceContentEntityStorage {
   public function __construct(EntityTypeInterface $entity_type, Connection $database, EntityManagerInterface $entity_manager, CacheBackendInterface $cache, LanguageManagerInterface $language_manager, EventDispatcherInterface $event_dispatcher, OrderRefreshInterface $order_refresh) {
     parent::__construct($entity_type, $database, $entity_manager, $cache, $language_manager, $event_dispatcher);
     $this->orderRefresh = $order_refresh;
+    $this->unchangedLoadIds = [];
```

Comment:
I just realized we can use a single flag for this, we don't need to track individual ids, they're processed one by one anyway. skipRefresh perhaps?